CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Added `MntFlags` and `unmount` on all of the BSDs.
 - Added `any()` and `all()` to `poll::PollFd`.
   ([#1877](https://github.com/nix-rust/nix/pull/1877))
+- Add `PF_ROUTE` to `SockType` on macOS.
+- Add `PF_ROUTE` to `SockType` on macOS, all of the BSDs and Linux.
```

Comment:
This is incorrect, because you _didn't_ add it to Linux.

Reply:
Ah, you are right. I've updated it now and I do apologize for the sloppy push.